core/src/test/java/org/apache/iceberg/TestMergeAppend.java

```diff
@@ -366,7 +366,8 @@ public void testChangedPartitionSpec() {
         1, base.currentSnapshot().manifests().size());
     ManifestFile initialManifest = base.currentSnapshot().manifests().get(0);
 
-    // build the new spec using the table's schema, which uses fresh IDs
+    // build the new spec using the table's sche
+    // ma, which uses fresh IDs
```

Comment:
Looks like this is an unnecessary change.

Reply:
done